EpikCord/__main__.py

```diff
@@ -1,4 +1,136 @@
-from EpikCord import __version__
+import argparse
+import os
+import re
+import sys
+import colorama
+from platform import system as get_parent_os
+
+__version__ = "0.5.2"
+_bot_tmplate = """#!/usr/bin/env python
+from EpikCord import Client, Intents
+from config import token
+class [name]Client(Client):
+
+    def __init__(self):
+        super().__init__()
+
+    #[_section]
+    
+intents = Intents.all()
+
+client: [name]Client = [name]Client(token, intents)
+
+
+@client.event()
+async def ready():
+    print("Ready!")
+# Write commands here
+
+client.login()
+"""
+__gitignore_template = """# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+# C extensions
+*.so
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+# Our configuration files
+config.py
+"""
+__config_file_template = """
+token = [token]
+"""
+if get_parent_os().lower() == "windows":
+    IS_WINDOWS = True
+else:
+    IS_WINDOWS = False
+
+
+class InvalidNameError(Exception):
+    ...
+
+
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--version", help="displays the version of the library", action="store_true"
+)
+parser.add_argument("-n","--new", help="Creates an new bot", action='store_true')
+
+colorama.init(autoreset=True)
+
+def get_question(name_query):
+        directory = input(
+            "Which directory to write the bot to?:(Type . to make a bot in the current directory): "
+        )
+        if not os.path.isdir(directory) or not directory == ".":
+            dir_question = input(
+                "This directory does not exist. Would you like me to create it?(y/n)"
+            )
+            if dir_question == "y" :
```

Comment:
Expect the user to ignore all direct instructions. Change this to `.startswith("y")`

Reply:
why?

Reply:
It's just good practise, always expect the user to be dumb, doesn't do the most basic stuff. Forms a good model

Reply:
it is literally clear that you need to type y or n

Reply:
But expect the user to ignore that. I don't think there is any word which implies an agreement that starts with `n`, and the other way round for no.